modules/management-console-agent/src/main/java/org/apache/ignite/agent/dto/action/RequestDeserializer.java

```diff
@@ -64,7 +67,7 @@ public RequestDeserializer() {
             }
         }
         catch (Exception e) {
-            req = new InvalidRequest().setCause(e).setId(id).setAction(act);
+            req = new InvalidRequest().setCause(new IllegalArgumentException(e)).setId(id).setAction(act);
```

Comment:
Why do we need to wrap exception with IllegalArgumentException?
And what if `e` already will be `IllegalArgumentException` (see line 57)?